packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -30,15 +30,25 @@ interface WatchPR {
   url: string;
 }
 
-handler.listPRs = async function listPRs(): Promise<WatchPR[]> {
+handler.getDatastore = async function getDatastore() {
   const query = datastore.createQuery(TABLE).order('created');
   const [prs] = await datastore.runQuery(query);
+  return [prs];
```

Comment:
Do we want to return an array here?  May be nice to add an explicit return type so we know this was on purpose (if that's the case)

Reply:
See comment below...Would love your help on it if you have any ideas!